Documentation/guides/messages/xa0035.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Xamarin.Android error XA0035
+description: XA0035 error code
+ms.date: 06/17/2020
+---
+# Xamarin.Android error XA0035
+
+## Issue
+
+Xamarin.Android was not able to determine the application's target
+[Android ABIs][abis] as specified by the `.csproj` file.
+
+[abis]: https://developer.android.com/ndk/guides/abis
+
+## Solution
+
+Make sure you are declaring valid values for `$(RuntimeIdentifiers)`:
```

Comment:
I think this wording is OK for now since, as far as I know, the full IDE experience is not yet know. Depending on how the IDE integration turns out, this can be updated to give a few more details about how to edit the `.csproj` file, like:

````
Open the project file [in Visual Studio][edit-project-files] or another text
editor and make sure all of the values in the `RuntimeIdentifiers` MSBuild
property are valid:

```xml
<PropertyGroup>
  <RuntimeIdentifiers>android.21-arm;android.21-arm64;android.21-x86;android.21-x64</RuntimeIdentifiers>
</PropertyGroup>
```

[edit-project-files]: https://docs.microsoft.com/visualstudio/msbuild/visual-studio-integration-msbuild#edit-project-files-in-visual-studio
````